avocado/core/loader.py

```diff
@@ -645,18 +645,24 @@ def _find_avocado_tests(self, path, class_name=None):
 
             # Looking for a 'class Anything(anything):'
             elif isinstance(statement, ast.ClassDef):
+
```

Comment:
I think the commit message is wrong (at least there is a typo, but I think the `disable` behavior changed...)